camera_matrix:
  cols: 3
  data:
  - 4773.22690159103
  - 0.0
  - 319.5
  - 0.0
  - 4773.22690159103
  - 239.5
  - 0.0
  - 0.0
  - 1.0
  rows: 3
camera_name: nuc3
distortion_coefficients:
  cols: 5
  data:
  - -0.4567835451282905
  - -265.713676850411
  - 0.004767632314955893
  - -0.001282046000798668
  - 0.0
  rows: 1
distortion_model: plumb_bob
image_height: 480
image_width: 640
projection_matrix:
  cols: 4
  data:
  - 4729.45703125
  - 0.0
  - 318.90124799479963
  - 0.0
  - 0.0
  - 4749.57470703125
  - 239.17123259041728
  - 0.0
  - 0.0
  - 0.0
  - 1.0
  - 0.0
  rows: 3
rectification_matrix:
  cols: 3
  data:
  - 1.0
  - 0.0
  - 0.0
  - 0.0
  - 1.0
  - 0.0
  - 0.0
  - 0.0
  - 1.0
  rows: 3
